src/test/java/org/carlspring/cloud/storage/s3fs/FileSystemProvider/CheckAccessTest.java

```diff
@@ -1,174 +0,0 @@
-package org.carlspring.cloud.storage.s3fs.FileSystemProvider;
```

Comment:
Could you please clarify why is this test case removed - is this covered somewhere else now?

Reply:
This wasn't removed, it was moved from "FileSystemProvider" folder to "fileSystemProvider" folder. That's why I think it is shown as deleted. This is the moved one:
https://github.com/carlspring/s3fs-nio/pull/83/files#diff-44f446394ced42e450b91bfc4e068d8d